src/nvim/msgpack_rpc/helpers.c

```diff
@@ -488,13 +488,20 @@ void msgpack_rpc_from_dictionary(Dictionary result, msgpack_packer *res)
   }
 }
 
-/// Handler executed when an invalid method name is passed
+
+/// No Op Handler used for Error handling
+Object msgpack_rpc_no_op(uint64_t channel_id,
```

Comment:
spell no-op as "noop":

    msgpack_rpc_noop

Reply:
okay, i will follow the convention then!